airbrake/notifier.py

```diff
@@ -83,6 +70,30 @@ def __init__(self, project_id=None, api_key=None, environment=None,
                 "by passing in the arguments explicitly."
             )
 
+        environment = config.get("environment",
+                                 os.getenv('AIRBRAKE_ENVIRONMENT'))
```

Comment:
Since this adds support for the many available options added to the README, it might be time to have a helper function that pulls configuration values from the environment, where any configurable value could be set by an environment variable. This would not only include airbrake credentials (e.g. api key) but also values that are sent in the payloads when notifying of an errors (e.g. environment, app_version, user_email). 

Something like

```python
def airbrake_environment_vars():
    return {k.lstrip('AIRBRAKE_').lower(): v
            for k, v in os.environ.items()
            if k.upper().startswith('AIRBRAKE_')}

It also seems reasonable to go forward with this PR, perhaps you could add this to the roadmap.